text/0000-else.md

```diff
@@ -0,0 +1,219 @@
+
+#### Code comments
+
+In the case of code comments, the behavior depends on whether `lwc:preserve-comments` is enabled or not. When _not_ preserving comments, comments may appear between conditional siblings:
+
+```html
+<!-- Valid -->
+<template>
+    <template if:true={foo}></template>
+    <!-- Comment! -->
+    <template lwc:else></template>
+</template>
+```
+
+Whereas when `lwc:preserve-comments` is enabled, comments become syntactically meaningful and therefore cannot be placed between sibling conditional directives:
+
+```html
+<!-- Invalid -->
+<template lwc:preserve-comments>
+    <template if:true={foo}></template>
+    <!-- Comment! -->
+    <template lwc:else></template>
+</template>
+```
+
+#### Accessors
+
+Similar to `if:true` and `if:false`, the expression passed in to `elseif:true` and `elseif:false` must use [simple dot notation](https://lwc.dev/guide/reference#if%3Atrue%7Cfalse%3D%7Bexpression%7D).
+
+#### Attribute values
+
+To avoid typos (e.g. using `lwc:else` when you mean `lwc:elseif`), `lwc:else` must _not_ have an attribute value. Any attribute value is treated as a compile-time error:
+
+```html
+<!-- Valid -->
+<template if:true={foo}></template>
+<template lwc:else></template>
+```
+
+```html
+<!-- Invalid -->
+<template if:true={foo}></template>
+<template lwc:else={yolo}></template>
+```
+
+Similarly, `elseif:true` and `elseif:false` _must_ have expressions as their values:
+
+```html
+<!-- Valid -->
+<template if:true={foo}></template>
+<template elseif:true={bar}></template>
+```
+
+```html
+<!-- Invalid -->
+<template if:true={foo}></template>
+<template elseif:true="bar"></template>
+```
+
+```html
+<!-- Invalid -->
+<template if:true={foo}></template>
+<template elseif:true></template>
+```
+
+### Semantics
+
+`elseif:true` and `elseif:false` behave the same as `if:true` and `if:false` respectively, except that their behavior depends on any preceding `if:true`/`if:false`/`elseif:true`/`elseif:false` statements. Similarly, the behavior of `lwc:else` also depends on its previous siblings.
+
+For example, in a series of sibling `if:` / `elseif:` / `else:` directives, the inverse of the previous evaluated expression determines whether we reach the next directive:
+
+```html
+<template if:true={abra}>
+    Abra!
+</template>
+<template elseif:true={kadabra}>
+    Kadabra!
+</template>
+<template elseif:false={hocus}>
+    Pocus!
+</template>
+<template lwc:else>
+    Kadabra!
+</template>
+```
+
+Rather than rehash the basics of conditional logic in programming, let's define this as being analogous to the [`if` statement](https://262.ecma-international.org/12.0/#prod-IfStatement) (and friends) in JavaScript:
+
+```js
+if (component.abra) {
+  // Abra!
+} else if (component.kadabra) {
+  // Kadabra!
+} else if (!component.hocus) {
+  // Pocus!
+} else {
+  // Kadabra!
+}
+```
+
+Note that this means:
+
+1. The property getters are only accessed once per definition in an `if:` or `ifelse:`.

```

Comment:
> per definition

I understand the overall meaning of this sentence, but per definition doesn't look like the correct term here.

Reply:
Changed to "per instance of."